airbyte-integrations/connectors/source-zapier-supported-storage/source_zapier_supported_storage/spec.yaml

```diff
@@ -0,0 +1,14 @@
+documentationUrl: https://docsurl.com
```

Comment:
update the doc to `https://docs.airbyte.com/integrations/sources/zapier-supported-storage`